cfgov/v1/util/util.py

```diff
@@ -79,9 +79,9 @@ def get_secondary_nav_items(request, current_page):
     nav_items = []
     for sibling in pages:
         if page.id == sibling.id:
-            sibling = page.get_appropriate_page_version(request)
```

Comment:
Is this `if/else` logic even needed anymore? In the first case (`page.id == sibling.id`) you're just assigning a page to itself. In the second case (`else`) you're doing the same thing.